5_databases/week2/week2-hmw-part4.sql

```diff
@@ -0,0 +1,76 @@
+-- Active: 1706786442782@@127.0.0.1@3306@Investment_Bank
+CREATE DATABASE Investment_Bank
+    DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE Investment_Bank;
+
+
+SELECT * FROM Client_Type;
+SELECT * FROM Registration_Number_Type;
+SELECT * FROM Instrument_Type;
+
+
+-- Create Client Type 
+CREATE TABLE Client_Type (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    client_type VARCHAR(255) NOT NULL
+);
+
+-- Create Table Registration Number Type
+CREATE TABLE Registration_Number_Type (
```

Comment:
can a client have multiple registration numbers?

Reply:
No, she shouldn't have.

Reply:
Awesome, you can reflect this in the same way as mentioned above. thank you :pray:

Reply:
Dropped 2 tiles and appended them to Client Table😊
Thank you for review❤️